docs/test-fairy/acct-mgmt/acct-settings.md

```diff
@@ -8,132 +8,136 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
-
-Global account settings can be managed on the **Preferences** page.
+You can manage the global account settings on the **Preferences** page.
 
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-1.png')} alt="Account Settings" width="600"/>
 
 The first two menu items are your SDK and API app tokens.
 
 ### SDK Token
+
 <br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-2.png')} alt="Account Settings" width="600"/>
 
-Your app token is used to initialize the [TestFairy SDK](https://docs.testfairy.com/SDK/Adding_The_Testfairy_SDK_To_Your_App.html).
+Your app token initializes the [TestFairy SDK](https://docs.testfairy.com/SDK/Adding_The_Testfairy_SDK_To_Your_App.html).
```

Comment:
Should this link go /test-fairy/sdk/adding-tf-sdk/?

Reply:
This page is not supported in sauce